openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -485,8 +484,7 @@ def login_refresh(request):
         return JsonResponse('Unauthorized', status=401)
 
     try:
-        response = JsonResponse({'success': True})
-        return refresh_jwt_cookies(request, response, request.user)
+        return get_response_with_refreshed_jwt_cookies(request=request, user=request.user)
```

Comment:
Nit: You don't really need the keywords for the arguments, and I find it actually makes this a little harder to read.